src/main/java/org/embulk/gradle/embulk_plugins/EmbulkPluginsPlugin.java

```diff
@@ -201,8 +202,15 @@ private static void configureAlternativeRuntimeDependencies(
                     if (projects.size() > 1) {
                         throw new GradleException("Multiple projects are found in dependency: " + dependency.toString());
                     }
-                    dependencies.add(project.getDependencies().create(
-                                         project.project(projects.iterator().next().getProjectPath())));
+                    final ProjectComponentIdentifier projectIdentifier = projects.iterator().next();
```

Comment:
may be `null` here?

Reply:
Ah, actually it must not happen (because `projects` must have at least one instance added when `types` has `"project"`), but it's too indirect. We may want to add a kind of assume...

Reply:
Added a check in 41e30129486dd17b5231aa93d4b0c0af62ff4631.